### DISCUSSION TITLE
Error updating component prop (out of nowhere)

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#8495] [Help]
All the sudden, I can't edit Javascript without getting 

"Error updating component prop"

whenever I go to save the component. It has never been an issue, and I haven't made any changes in the past few days. 

Just logged in today, this is the first thing I've tried, and it's out of nowhere. I also tried simply updating components and the same error comes up. It seems I can't update anything. 

I'm on Budibase cloud.

**Comment by stellarfinn** [comment]
Update: It appears to only happen on some screens of my app and not others. To reiterate, I've changed nothing since the last time it was working.

**Comment by fcapurso** [comment]
I'm having the same issue, pretty much with anything I want to change in the design... initially I thought it was related to upgrading to 2.1.8 but I reverted back to 2.1.1 and also reverted all changes to the latest published version (the only changes I was able to make were to delete some screens).
Still having the same issue, while yesterday evening (with same budibase and app versions) everything was working fine...

**Reply by stellarfinn** [reply]
My problem as well. I even exported the app, imported it as a new app, same issue. I also can't duplicate the affected screens. The screens are basically stuck... can't make any changes to them... even simple ones like changing title text or something.

**Reply by fcapurso** [reply]
Issue is fixed for me in release 2.1.14.

**Comment by ghost** [comment]
I'm having this issue too. Reported in on #8483.
